datastore/save.go

```diff
@@ -53,6 +54,32 @@ func reflectFieldSave(props *[]Property, p Property, name string, opts saveOpts,
 	switch x := v.Interface().(type) {
 	case *Key, time.Time, GeoPoint:
 		p.Value = x
+	case civil.Date, civil.DateTime, civil.Time:
+		if dateVal, ok := v.Interface().(civil.Date); ok {
+			p.Value = dateVal.In(time.Now().Location())
```

Comment:
Seems like this should always use UTC?

Reply:
After overthinking this once again, yeah, I think it should.